src/entrypoint.scss

```diff
@@ -3,17 +3,14 @@
 
 // This adds the background
 html::after {
-  content: "";
-  background: url(images/rwb.svg) no-repeat center center fixed;
-  -webkit-background-size: cover;
-  -moz-background-size: cover;
-  -o-background-size: cover;
+  background: url('images/rwb.svg') no-repeat center center fixed;
   background-size: cover;
-  opacity: 0.5;
-  top: 0;
-  left: 0;
   bottom: 0;
-  right: 0;
+  content: '';
+  left: 0;
+  opacity: .5;
   position: absolute;
+  right: 0;
+  top: 0;
```

Comment:
I know I mentioned the no units thing before...now I'm concerned I've botched it all.